### PR TITLE
A kit to help new botanists discover chemistry

### PR DESCRIPTION
**Pull request by drakewill-CRL**
## About the PR

Added the "Agrichem is Fun!" kit to botany's Nutrimax vendor. It's a box containing 30u of several chemicals to make basic fertilizers, and 5u of 2 others to make unstable mutagen. It's meant to be reminiscent of the old 1950s chemistry kits that had uranium or other dangerous chemicals in them, and useful for new botanists.

## Why / Balance

Hydroponics is a pretty chill department, but there's very little in-game information or training on anything there. A lot of learning comes from the wiki or other players. This change is done to provide something in-game that shows new botanists there's more to the job than "plant seed in tray, harvest produce later". It also introduces them to the game's chemistry system with some good chemicals they will want and an explanation of what they do. 

Yes, all this information is in the guidebook. No, not everyone is going to read that. This is a way to find some of that out in-game without reading the OOC guidebook or wiki or such. 

This kit is pretty limited, both in amount of reagents and products you can make with them. The idea is to get more of the people learning botany to look at the next step in complexity on the job, since there's not much right now to help you discover that on your own. The kit shows you how to make more of the baseline chemicals you use, along with alternative chemicals with stronger effects, and a little bit (15u) of robust mutagen without bugging chemistry. 

## Technical details

This is all YAML, no C#.

## Media

Warning: codersprites

image

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

:cl:
- add: Botanists can now learn basic chemistry with the Agrichem Is Fun! kit in the Nutrimax vendor.

### CODE REVIEW

**Comment by Emisse** [comment]
this is cute

File: Resources/Prototypes/Catalog/Fills/Boxes/service.yml [hunk 1]
```diff
@@ -11,4 +11,26 @@
   - type: Sprite
     layers:
       - state: box
-      - state: flashbang
\ No newline at end of file
+      - state: flashbang
+
+- type: entity
+  name: AgriChem Is Fun! kit
```
Comment (lzk228) [inline Resources/Prototypes/Catalog/Fills/Boxes/service.yml:17]: maybe change it to `"AgriChem Is Fun" kit`? and the same with manual `"AgriChem Is Fun" manual`

Comment (drakewill-CRL) [reply Resources/Prototypes/Catalog/Fills/Boxes/service.yml]: Apparently you can only use quotes in the name if they're not the first character, so I've changed them to `NT "AgriChem is Fun" kit`

**Comment by Plykiya** [comment]
mandatory feature

File: Resources/Locale/en-US/paper/paper-misc.ftl [hunk 2]
```diff
@@ -27,6 +27,29 @@ book-text-ame-scribbles = I don't know if you're trained already, so I hope this
...
+      When you're ready to show everyone you're the greatest botanist on the station, you'll need to mix up some Farmer's Helper(TM) and hope for the best. Farmer's Helper(TM) is a newly discovered fertilizing agent powered by the element of the modern age: radium!
+      Mix equal parts radium, phosphorus, and chlorine together to get a bright green batch of Farmer's Helper(TM), the safe and fun way to get the most of your farm. The 15u this kit lets you make could be your introduction to agrichemical stardom!
+      Farmer's Helper(TM) can have a wide variety of effects on plant life, including drastic changes to all sorts of growth parameters, making plants glow in the dark, or creating entirely new species.
+      Each individual plant responds to Farmers Helper(TM) differently, so you may want to use small doses on multiple crops and try to crossbreed the best traits from each of those.
+      Farmer's Helper(TM) is a patended and entirely safe fertilizer, and NanoTrasen takes no responsibility for dead crops, excessive water bills, newly sentient plants asking existential questions, or flora-strangled farmhands that may coincidentally occur while using it.
```
Comment (deltanedas) [inline Resources/Locale/en-US/paper/paper-misc.ftl:51]: patended
whos pat

**Comment by slarticodefast** [comment]
These are so much fun to read. You are great at writing :)
Thank you for your contribution!

File: Resources/Locale/en-US/paper/paper-misc.ftl [hunk 3]
```diff
@@ -27,6 +27,32 @@ book-text-ame-scribbles = I don't know if you're trained already, so I hope this
...
+      Unstable mutagen can have a wide variety of effects on plant life, including drastic changes to all sorts of growth parameters, produce full of helpful pharmaceuticals, plants that glow in the dark, or creating entirely new species.
+      
+      Each individual plant responds to unstable mutagen differently, so you may want to use small doses on multiple crops and try to crossbreed the best traits from each of those. Applying multiple doses to one plant can stack multiple changes and make it harder to single out desirable traits.
+      Unstable mutagen is entirely safe when used as a fertilizer, and NanoTrasen takes no responsibility for dead crops, excessive water bills, newly sentient plants asking existential questions, or flora-strangled farmhands that may coincidentally occur while using it.
+      Do not drink unstable mutagen. Wash your hands thoroughly after handing. Wash your eyes if you have looked at unstable mutage for over 30 minutes in a 24 hour period. Store in a dark room between 293–295K. Do not use on corporate holidays. If you begin hearing voices telling you to drink unstable mutagen, please contact your doctor, head of personnel, or exorcist.
```
Comment (perryprog) [inline Resources/Locale/en-US/paper/paper-misc.ftl:54]:
```suggestion
      Do not drink unstable mutagen. Wash your hands thoroughly after handing. Wash your eyes if you have looked at unstable mutagen for over 30 minutes in a 24 hour period. Store in a dark room between 293–295K. Do not use on corporate holidays. If you begin hearing voices telling you to drink unstable mutagen, please contact your doctor, head of personnel, or exorcist.
```

Comment (slarticodefast) [reply Resources/Locale/en-US/paper/paper-misc.ftl:54]: Damn, I missed it. Can someone do a 1 char fix PR?

**Pull request merged**